### DISCUSSION TITLE
Running composer without loading unecessary images

### DISCUSSION THREAD

**Discussion by ivancarlosti** [pglombardo/PasswordPusher#2162] [Q&A]
I am running PasswordPusher by docker-compose, and I noticed that it loads several stuffs like mariadb-dev sqlite-dev...
How can I run compose without adding these packages on my image built?
I already removed db from my .yml file because I run my own mariadb database.

![image](https://github.com/pglombardo/PasswordPusher/assets/15795767/bc918b88-a99a-4166-8cef-5da478c7174a)

Thanks!!

**Comment by pglombardo** [comment]
Hi @ivancarlosti - the Password Pusher docker container bundles those libraries so that it can optionally connect to either.

Some history is that I used to offer 3 individual containers `pwpush-ephemeral` (SQLite), `pwpush-mysql` and `pwpush-postgres` but maintenance and user confusion began to be too much so we switched to a single universal container.

If you really wanted, you could just remove those packages from the apk line in the Dockerfile.  But if you do that, you'll have to update the source code by removing the entries from `Gemfile` and re-running `bundle install`.  Having to do that every release is not worth it IMO but FYI.
